airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -121,6 +123,28 @@ public abstract class DestinationAcceptanceTest {
    */
   protected abstract String getImageName();
 
+  private String getImageNameWithoutTag() {
+    return getImageName().contains(":") ? getImageName().split(":")[0] : getImageName();
+  }
+
+  private Optional<StandardDestinationDefinition> getOptionalDestinationDefinitionFromProvider(final String imageNameWithoutTag) {
+    try {
+      LocalDefinitionsProvider provider = new LocalDefinitionsProvider(LocalDefinitionsProvider.DEFAULT_SEED_DEFINITION_RESOURCE_CLASS);
```

Comment:
this means that the destination acceptance tests now depend on the destination_definitions.yaml file - is that ok? (I don't know if this explicitly breaks anything, just want to doublecheck that we're not introducing any exciting circular dependencies. E.g. if I have a PR that modifies some behavior in `destination-foo` _and_ switches it from `normalization` to `normalization-foo`, is that still easy to do?)

Reply:
Yes, you are right that for destination acceptance tests we end up depending on `destination_definitions.yaml`. As for your example, switches `normalization` to `normalization-foo` for `destination-foo` is very easy. Only one field for destination-foo in the `destination_definitions.yaml` file will need to be updated. This is the easiest way to integrate new logic into destination acceptance tests.